lib/DirectEditing.js

```diff
@@ -44,10 +44,12 @@ DirectEditing.prototype.registerProvider = function(provider) {
 /**
  * Returns true if direct editing is currently active
  *
- * @return {Boolean}
+ * @param {djs.model.Base} element
```

Comment:
I would mark the parameter as optional in JSDocs. Other than that, ready to go :rocket: 
```suggestion
 * @param {djs.model.Base} [element]
```